### PR TITLE
Revert Portable/CodeWarrior formatting

### PR DESCRIPTION
**Pull request by Skptak**
Description
-----------
Revert the formatting on CodeWarrior ports

Test Steps
-----------

Checklist:
----------

- [ ] I have tested my changes. No regression in existing tests.
- [ ] I have modified and/or added unit-tests to cover the code changes in this Pull Request.

Related Issue
-----------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: portable/CodeWarrior/ColdFire_V1/portmacro.h [hunk 1]
```diff
@@ -29,11 +29,9 @@
 #ifndef PORTMACRO_H
 #define PORTMACRO_H
 
-/* *INDENT-OFF* */
 #ifdef __cplusplus
-    extern "C" {
+extern "C" {
 #endif
-/* *INDENT-ON* */
```
Comment (chinglee-iot) [inline portable/CodeWarrior/ColdFire_V1/portmacro.h:36]: Suggest we add back the INDENT-OFF/ON comments.

File: portable/CodeWarrior/ColdFire_V1/portmacro.h [hunk 2]
```diff
@@ -46,34 +44,32 @@
...
+
+
+#if( configUSE_16_BIT_TICKS == 1 )
+    typedef uint16_t TickType_t;
+    #define portMAX_DELAY ( TickType_t ) 0xffff
 #else
-    #error configTICK_TYPE_WIDTH_IN_BITS set to unsupported tick type width.
+    typedef uint32_t TickType_t;
+    #define portMAX_DELAY ( TickType_t ) 0xffffffffUL
 #endif
```
Comment (chinglee-iot) [inline portable/CodeWarrior/ColdFire_V1/portmacro.h:66]: configTICK_TYPE_WIDTH_IN_BITS is also wrongly reverted. We should add it back.

File: portable/CodeWarrior/ColdFire_V1/portmacro.h [hunk 3]
```diff
@@ -83,47 +79,34 @@ uint32_t ulPortSetIPL( uint32_t );
...
-            traceISR_EXIT_TO_SCHEDULER();        \
-            portYIELD();                         \
-        }                                        \
-        else                                     \
-        {                                        \
-            traceISR_EXIT();                     \
-        }                                        \
-    } while( 0 )
-
-/* *INDENT-OFF* */
```
Comment (chinglee-iot) [inline portable/CodeWarrior/ColdFire_V1/portmacro.h:123]: Same suggestion for the INDENT-OFF/ON

File: portable/CodeWarrior/ColdFire_V2/portmacro.h [hunk 4]
```diff
@@ -29,11 +29,9 @@
 #ifndef PORTMACRO_H
 #define PORTMACRO_H
 
-/* *INDENT-OFF* */
```
Comment (chinglee-iot) [inline portable/CodeWarrior/ColdFire_V2/portmacro.h:32]: Same suggestion for INDENT-OFF/ON

File: portable/CodeWarrior/ColdFire_V2/portmacro.h [hunk 5]
```diff
@@ -46,33 +44,31 @@
...
-#define portLONG          long
-#define portSHORT         short
-#define portSTACK_TYPE    uint32_t
-#define portBASE_TYPE     long
-
-typedef portSTACK_TYPE   StackType_t;
-typedef long             BaseType_t;
-typedef unsigned long    UBaseType_t;
-
-#if ( configTICK_TYPE_WIDTH_IN_BITS == TICK_TYPE_WIDTH_16_BITS )
```
Comment (chinglee-iot) [inline portable/CodeWarrior/ColdFire_V2/portmacro.h:61]: Same suggestion for configTICK_TYPE_WIDTH_IN_BITS.

File: portable/CodeWarrior/ColdFire_V2/portmacro.h [hunk 6]
```diff
@@ -81,48 +77,35 @@ uint32_t ulPortSetIPL( uint32_t );
...
-        }                                        \
-    } while( 0 )
-
-/* *INDENT-OFF* */
+#define portEND_SWITCHING_ISR( xSwitchRequired )    do { if( xSwitchRequired != pdFALSE ) { portYIELD(); } } while( 0 )
+
+
 #ifdef __cplusplus
-    }
+}
```
Comment (chinglee-iot) [inline portable/CodeWarrior/ColdFire_V2/portmacro.h:108]: INDENT-OFF/ON suggestion.

File: portable/CodeWarrior/HCS12/portmacro.h [hunk 7]
```diff
@@ -41,70 +41,68 @@
...
+#define portLONG        long
+#define portSHORT       short
+#define portSTACK_TYPE  uint8_t
+#define portBASE_TYPE   char
+
+typedef portSTACK_TYPE StackType_t;
+typedef signed char BaseType_t;
+typedef unsigned char UBaseType_t;
+
+#if( configUSE_16_BIT_TICKS == 1 )
```
Comment (chinglee-iot) [inline portable/CodeWarrior/HCS12/portmacro.h:56]: configTICK_TYPE_WIDTH_IN_BITS suggestion.

**Review by chinglee-iot — CHANGES_REQUESTED** [review]
Compared with commit before #768 
d6bccb1f4 | 2023-09-05 | Gaurav-Aggarwal-AWS | Fix heap address calculation issue (#781)

https://github.com/FreeRTOS/FreeRTOS-Kernel/compare/d6bccb1f4...Skptak:FreeRTOS-Kernel:fixCodeWarriorFormatting

* INDENT-OFF/ON comments are removed. Suggest we add them back.
* configTICK_TYPE_WIDTH_IN_BITS is wrongly reverted. We should add it back.

**Review by chinglee-iot — APPROVED** [review]

**Review by aggarg — APPROVED** [review]

**Pull request merged**